{"name": "测试正确账号、错误密码情况下的登录", "status": "failed", "statusDetails": {"message": "AssertionError: assert '账号或密码不正确' == '账号或密码1不正确'\n  \n  - 账号或密码1不正确\n  ?      -\n  + 账号或密码不正确", "trace": "self = <test_001_loginfail.TestLoginFail object at 0x0000029626787FD0>, driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"090b2916283602fbd02ff889e2ab8dbc\")>, base_url = 'http://10.88.40.127:23010'\n\n    @allure.title('测试正确账号、错误密码情况下的登录')\n    @allure.description('登录失败：测试正确账号、错误密码登录')\n    def test_login_fail_case(self, driver, base_url):\n        \"\"\"\n        名称：使用正确账密登录\n        步骤：\n        1、打开浏览器\n        2、输入正确账号，错误密码\n        3、点击登录按钮\n        检查点：\n        * 检查页面是否提示“账号或密码不正确”。\n        \"\"\"\n        page = LgPage(driver)\n        page.open(base_url)\n    \n        # 显示等待提成了公共方法\n        waits.waits(driver, \"ID\", \"login_userName\", \"qianchuan\")\n        #显示等待\n        # myelement = WebDriverWait(driver,5,0.5).until(\n        #     EC.visibility_of_element_located((By.ID,\"login_userName\"))\n        # )\n        # myelement.send_keys('qianchuan')\n    \n        #隐式等待\n        # driver.implicitly_wait(10)\n    \n        # 因为显示等待和PO的冲突 第一个元素就不使用PO模式了\n        # page.login_account = \"qianchuan1\"\n        page.login_pwd = \"Qianchuan@12345\"\n        page.login_code = \"1111\"\n        page.login_login_button.click()\n    \n        sleep(0.5)\n    \n        # 调试：打印 login_error_message 的类型和值\n        print(\"Type of login_error_message:\", type(page.login_error_message))\n        print(\"Type of .text:\", type(page.login_error_message.text))\n    \n>       assert page.login_error_message.text == \"账号或密码1不正确\"\nE       AssertionError: assert '账号或密码不正确' == '账号或密码1不正确'\nE         \nE         - 账号或密码1不正确\nE         ?      -\nE         + 账号或密码不正确\n\ntest_dir\\Login\\test_001_loginfail.py:55: AssertionError"}, "description": "登录失败：测试正确账号、错误密码登录", "attachments": [{"name": "stdout", "source": "cedc1836-81a2-4871-9c7f-315a742b7396-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "ddb792e4-b546-409e-8600-26620fafe74b-attachment.txt", "type": "text/plain"}], "start": 1749455220041, "stop": 1749455227365, "uuid": "0afc3908-74dd-46e5-a60c-93234da2d7cf", "historyId": "9bafb470a3f5b2bc06ae6138b0479d4f", "testCaseId": "9bafb470a3f5b2bc06ae6138b0479d4f", "fullName": "test_dir.Login.test_001_loginfail.TestLoginFail#test_login_fail_case", "labels": [{"name": "feature", "value": "登录失败"}, {"name": "suite", "value": "登录"}, {"name": "parentSuite", "value": "test_dir.Login"}, {"name": "subSuite", "value": "TestLoginFail"}, {"name": "host", "value": "DESKTOP-9JRI8LF"}, {"name": "thread", "value": "3728-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_dir.Login.test_001_loginfail"}]}